src/D2L.Bmx/Browser.cs

```diff
@@ -39,24 +35,27 @@ public static class Browser {
 		return await Puppeteer.LaunchAsync( launchOptions );
 	}
 
-	private static string? GetPathToBrowser() {
+	public static bool TryGetPathToBrowser( [NotNullWhen( returnValue: true )] out string? path ) {
+		path = null;
```

Comment:
Changes in this method are not needed for the purpose of this task. Just a style change to make it more conventionally C#.